intellij/src/saros/intellij/runtime/IntelliJSynchronizer.java

```diff
@@ -32,7 +36,11 @@ private void exec(Runnable runnable, boolean async) {
     if (async) {
       application.invokeLater(runnable, ModalityState.defaultModalityState());
     } else {
-      application.invokeAndWait(runnable, ModalityState.defaultModalityState());
+      try {
+        application.invokeAndWait(runnable, ModalityState.defaultModalityState());
+      } catch (ProcessCanceledException e) {
+        log.error("Synchronous execution on EDT interrupted - " + runnable, e);
```

Comment:
What is interrupted ? The waiting for the execution to complete or the execution itself ?

Reply:
The [documentation of the `incokeAndWait(...)`](https://github.com/JetBrains/intellij-community/blob/master/platform/core-api/src/com/intellij/openapi/application/Application.java#L277) call says the following:
```Java
@throws ProcessCanceledException when the current thread is interrupted
```
So I...I guess both? Its not really specific...